assets/constants/OnboardingSteps.js

```diff
@@ -0,0 +1,118 @@
+/*
+--------------------------------------------------------------
+| PLEASE READ IF YOU ARE NEW TO THIS SECTION OF THE CODEBASE |
+--------------------------------------------------------------
+Be very careful when adding a new step. You could easily break the whole system if you do not update all the steps
+affected by the creation of a new one (Usually existing steps after the newly created one)
+
+You will also have to adjust the indexes of all the terms in all the locales files.
+
+So in otherwords don't add new steps unless you are sure you really really need to.
+*/
+
+import NavigationService from '../../utils/NavigationService';
+import { COMPLETED_KEYS } from '../../actions/onboarding';
+
+export const ORDERED_STEPS = [
+  'zero',
+  'welcomeScreen', // Welcome user to app
+  'addLocation', // Tell user how to add location
+  'addLocationAction', // Wait for location to be created
+  'viewPinCallout', // Tell user to click on pin
+  'viewPinCalloutAction', // Wait for user to click pin
+  'pinCalloutDescription', // Explain pin callout
+  'invitations1', // Inform user that invitation needed to upload
+  'invitations2', // Open side drawer
+  'invitations3', // Click Accept Invite button
+  'invitations4', // Explain how to accept invite using QR scanner
+  'invitationsAction', // Wait for user to accept a valid invite
+  'invitationAccepted', // Explain what happens now that they have accepted invite
+  'endScreen', // Say closing remarks
```

Comment:
For another PR, we were talking how it might be good to include a prompt letting the user know they can move a pin while they are creating it.

For the initial suggestion you for creating a pin Matt was thinking something like this:

![image](https://user-images.githubusercontent.com/103917/46512778-b9b72c00-c823-11e8-8f0a-24e74324e704.png)